authenticator/babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function(api) {
+  api.cache(true);
+  return {
+    presets: ['babel-preset-expo'],
```

Comment:
I looked this up, and this seems like a really great addition to presets :)

Reply:
That was part of expo's default blank project initializer